src/java/com/wolfssl/provider/jsse/WolfSSLImplementSSLSession.java

```diff
@@ -77,6 +77,11 @@ public class WolfSSLImplementSSLSession implements SSLSession {
      * setResume() or finalization.
      */
     protected boolean isInTable = false;
+
+    /**
+     * Tracks if WOLFSSL_SESSION poiner has been updated after retreived from
```

Comment:
typo 'poiner'

Reply:
Thanks! Fixed and squashed into last commit.